backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/objcexport/ObjCExportCodeGenerator.kt

```diff
@@ -1056,7 +1054,7 @@ private fun ObjCExportCodeGenerator.createObjectInstanceAdapter(
         descriptor: ClassDescriptor
 ): ObjCExportCodeGenerator.ObjCToKotlinMethodAdapter {
     assert(descriptor.kind == ClassKind.OBJECT)
-    assert(!descriptor.isUnit())
+    assert(!descriptor.isUnit() && !descriptor.isNothing())
```

Comment:
This code handles `descriptor.kind == ClassKind.OBJECT` case.
Is there `object Nothing`?